packages/product-components/src/components/TopAssets/TopAssets.tsx

```diff
@@ -43,12 +50,12 @@ export interface TopAssetsProps {
     assets: Asset[];
     onAssetClick: (asset: Asset) => void;
     logoSize?: AssetLogoProps['size'];
-    className?: string;
+    dataTestId?: string;
```

Comment:
Still don't agree with this one, don't understand why you insist on this inconsistency so much, but I can refactor it later...